internal/init_test.go

```diff
@@ -13,3 +17,25 @@ func TestAccBundleInitErrorOnUnknownFields(t *testing.T) {
 	_, _, err := RequireErrorRun(t, "bundle", "init", "./testdata/init/field-does-not-exist", "--output-dir", tmpDir)
 	assert.EqualError(t, err, "failed to compute file content for bar.tmpl. variable \"does_not_exist\" not defined")
 }
+
+func TestAccBundleInitOnMlopsStacks(t *testing.T) {
```

Comment:
Would it be possible to deploy this stack to make sure it works as expected?

Reply:
Done